### PR TITLE
This change addresses following issues:

### PR DESCRIPTION
**Pull request by rhim**
1. Removes port information from Admin guide, adds it to concepts guide
2. Adds support and validation matrix
3. Removes api reference and points it to a WIP page, changes
CONTRIBUTING.md about api spec
4. Fixes contiv-compose tutorial
5. Other minor cleanups

Fixes contiv/ccn#275

Signed-off-by: Himanshu Raj <rhim@users.noreply.github.com>

### CODE REVIEW

File: websrc/source/documents/tutorials/contiv-compose.md [hunk 1]
```diff
@@ -6,33 +6,42 @@ description: |-
   Getting Started
 ---
 
-# Policies with Networking
+## Networking Policies with Compose
```
Comment (gaurav-dalvi) [inline websrc/source/documents/tutorials/contiv-compose.md]: with `docker-compose`

File: websrc/source/documents/tutorials/contiv-compose.md [hunk 2]
```diff
@@ -44,7 +53,7 @@ $ cd $GOPATH/src/github.com/contiv/netplugin
 $ make ssh
 ```
 
-### 3. Compile the Software
+#### Step 4: Compile the Software
```
Comment (gaurav-dalvi) [inline websrc/source/documents/tutorials/contiv-compose.md]: How to compile this ?

File: websrc/source/documents/tutorials/index.md [hunk 3]
```diff
@@ -9,6 +9,6 @@ description: |-
 # Tutorials
 
 - [Container Networking Tutorial](/documents/tutorials/container-101.html)
-- [Contiv Policy Networking Tutorial](/documents/tutorials/contiv-policy.html)
-- [Policies with Networking](/documents/tutorials/contiv-compose.html) [Deprecated]
+- [Contiv Policy Tutorial](/documents/tutorials/contiv-policy.html)
+- [Networking Policies with Compose](/documents/tutorials/contiv-compose.html)
```
Comment (gaurav-dalvi) [inline websrc/source/documents/tutorials/index.md]: docker-compose

**Comment by gaurav-dalvi** [comment]
@rhim: you have hardly 3-4 md file change. How come you are editing 61 files with this change ?, Did you run the site on local environment and browsed through it.

File: websrc/source/documents/tutorials/contiv-compose.md [hunk 4]
```diff
@@ -6,33 +6,42 @@ description: |-
...
 
-- Make
-- Git
+### Setup
 
-### 1. Start the Vagrant Environment
-Use the following commands to start a Contiv Vagrant setup.
+#### Step 1: Get contiv installer code from github.
+```
+$ git clone git@github.com:contiv/install.git
```
Comment (dseevr) [inline websrc/source/documents/tutorials/contiv-compose.md]: Can you use the https URL here?  `git@...` is SSH which requires a github account + ssh keypair setup.  https will work for anyone

**Comment by dseevr** [comment]
@gaurav-dalvi the edit to `layouts/documents.erb` meant that every page with the sidebar needed its HTML regenerated

File: websrc/source/documents/tutorials/contiv-compose.md (hunk 4 above)
Comment (dseevr) [inline websrc/source/documents/tutorials/contiv-compose.md]: `git clone https://github.com/contiv/install.git`

**Pull request merged**